scripts/tools/zap/version_update.py

```diff
@@ -51,6 +51,7 @@
 USAGE_FILES_DEPENDING_ON_ZAP_VERSION = [
     'integrations/docker/images/chip-cert-bins/Dockerfile',
     'scripts/setup/zap.json',
+    'scripts/setup/zap.version',
```

Comment:
zap version is already in zap.json. Could we parse/process json instead? Why keep the same information in 2 separate files?